src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  $cell-side: 100px;
+  $borders: 1px;
+  max-width: 8 * $cell-side;
+  padding: $cell-side;
+  display: flex;
+  gap: $borders;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    height: $cell-side;
+    width: $cell-side;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: $borders solid black;
+    transition: transform 0.5s;
```

Comment:
What about background-color? You change it on hover too.
Also, would be great to create a variable for transition-duration
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/990128ac-e93f-4d5d-b801-9300295fa755)